src/util.jl

```diff
@@ -22,16 +22,15 @@ end
     return floor(Int, i)
 end
 
-abstract type AbstractUserBackend end
+abstract type AbstractThreadingBackend end
 
 """
     PolyesterBackend()
 
-A type to specify the parallelization backend when using [`@threaded`](@ref). If this type
-is specified, the @threaded macro will use `Polyester.@batch` for the multithreaded `for`
-loop.
+Pass as first argument to the [`@threaded`](@ref) macro to make the loop multithreaded
+with `Polyester.@batch`.
```

Comment:
Can you please change the other two docstrings as well?

Reply:
Sorry, I forgot to change them - fixed it.